src/components/legend/style.js

```diff
@@ -472,3 +487,7 @@ module.exports = function style(s, gd) {
         }
     }
 };
+
+function getGradientDirection(reversescale) {
```

Comment:
Looks like `reversescale` is only used as an argument for this function. If that's the case, we should :hocho: this and have

```js
var cOpts = extractOpts(trace);
var colorscale = cOpts.colorscale;
var gradientDirection = cOpts.reversescale ? 'horizontal' : 'horizontalreversed';
```

Reply:
True. But soon it would be used in other places when adding new legends 3-D.

Reply:
`getGradientDirection` or `reversescale` will get used?

Reply:
Both.

Reply:
All right - if you say so. Closing this comment, but please add a commit for https://github.com/plotly/plotly.js/pull/4356#discussion_r347573960